contracts/BurnExecution.sol

```diff
@@ -95,51 +93,61 @@ contract BurnExecution is FraudProofHelpers {
         return (stateRoot, actualTxRoot, !isTxValid);
     }
 
+    function ApplyBurnExecutionTx(
+        Types.AccountMerkleProof memory _fromAccountProof,
+        Types.BurnExecution memory _tx
+    ) public view returns (bytes memory updatedAccount, bytes32 newRoot) {
+        Types.UserAccount memory account = _fromAccountProof.accountIP.account;
+
+        account.balance -= account.burn;
+        account.burn = 0;
+        account.lastBurn = RollupUtils.GetYearMonth();
+
+        updatedAccount = RollupUtils.BytesFromAccount(account);
+        newRoot = UpdateAccountWithSiblings(account, _fromAccountProof);
+        return (updatedAccount, newRoot);
+    }
+
     /**
      * @notice Overrides processTx in FraudProof
      */
-    function processTx(
+    function processBurnExecutionTx(
         bytes32 _balanceRoot,
-        bytes32 _accountsRoot,
         Types.BurnExecution memory _tx,
-        Types.PDAMerkleProof memory _from_pda_proof,
-        Types.AccountProofs memory accountProofs
+        Types.AccountMerkleProof memory _fromAccountProof
```

Comment:
@vaibhavchellani I removed pda proof and the pubkey tree root for now. We might want to add it back depending on how we do the signature validation.